### DISCUSSION TITLE
Batched faster-whisper sometimes cannot capture sentences in an audio

### DISCUSSION THREAD

**Discussion by toanhn** [SYSTRAN/faster-whisper#1001] [General]
Batched faster-whisper sometimes cannot capture sentences in an audio though the original faster-whisper can.
Can you please help me to deal with this issue, thanks.

**Comment by MahmoudAshraf97** [comment]
use this branch #936

**Reply by toanhn** [reply]
I used this command to install batched faster-whisper on my local PC: pip install --force-reinstall "faster-whisper @ https://github.com/SYSTRAN/faster-whisper/archive/refs/heads/master.tar.gz"

Can you provide me the command to install this branch https://github.com/SYSTRAN/faster-whisper/pull/936 in my local PC?
Thank you so much.

**Reply by MahmoudAshraf97** [reply]
`pip install git+https://github.com/MahmoudAshraf97/faster-whisper.git@same_vad`

**Reply by toanhn** [reply]
@MahmoudAshraf97  Thank you so much, friend.
